packages/orchestrator-ui-components/src/pages/workflow/WFOStartWorkflowPage.tsx

```diff
@@ -22,17 +38,25 @@ export interface UserInputForm {
 export const WFOStartWorkflowPage = ({
     workflowName,
     productId,
+    subscriptionId,
 }: WFOStartWorkflowPageProps) => {
     const apiClient = useAxiosApiClient();
+    const t = useTranslations('processes.steps');
+    const router = useRouter();
+    const { theme } = useOrchestratorTheme();
     const [isFetching, setIsFetching] = useState<boolean>(true);
     const [form, setForm] = useState<UserInputForm>({});
     const { stepUserInput, hasNext } = form;
+    const { stepHeaderStyle, stepListContentBoldTextStyle } = getStyles(theme);
 
     const submit = useCallback(
         (processInput: object[]) => {
             if (workflowName && productId) {
                 processInput.unshift({ product: productId });
             }
+            if (workflowName && subscriptionId) {
+                processInput.unshift({ subscription_id: subscriptionId });
```

Comment:
Might be personal preference, but I would like to suggest to use the spread operator over changing the incoming array itself.
const updatedProcessInput = [{ subscription_id: subscriptionId }, ...processInput]

Reply:
This doesn't work because it's inside a if/else block unless you make it a let. Made a function that does it that solves the problem